arkouda/pdarrayclass.py

```diff
@@ -1217,6 +1217,35 @@ def list_registry() -> List[str]:
 
     return registered_list
 
+def list_symbol_table() -> List[str]:
+    """
+    Return a list containing the names of all objects in the symbol table
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the symbol table
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    symbol_list: List[str] = []
+
+    if info(AllSymbols) not in [EmptyRegistry, EmptySymbolTable]:
+        symbol: str
+        for symbol in filter(None, info(AllSymbols).split('\n')):
+            if symbol is not None:
```

Comment:
You shouldn't need this guard condition since you are filtering out `None` in the loop generator.

Reply:
we chatted offline. I think I originally had just the filter and mypy was not happy. But there's no point in double checking, so I'll remove the filter and leave the guard condition